modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/index.js

```diff
@@ -184,6 +187,17 @@ const Renderer = ({
 		PAGE_HEADER_COMPONENT_TYPE[page.headerRenderer] ||
 		Components.PageHeader;
 
+	let hasFieldRequired = false;
+	const visitor = new PagesVisitor(pages);
+	visitor.mapFields((field) => {
+		if (field.required) {
+			hasFieldRequired = true;
+		}
+	});
+	const isAdminPortletNamespace = portletNamespace.includes('Admin');
+	const hasRequiredIconDescription =
```

Comment:
@georgel-pop-lr, do we really need this variable? Let's use only `hasFieldRequired` and `isAdminPortletNamespace` in the check of line 232 following this alphabetic order (`hasFieldRequired` and `isAdminPortletNamespace`). 🙂

Reply:
@natocesarrego Change done please check. Regards.